cmd/src/bin/tikv-ctl.rs

```diff
@@ -1777,6 +1798,43 @@ fn main() {
         return;
     }
 
+    if let Some(matches) = matches.subcommand_matches("decrypt-file") {
+        let infile = matches.value_of("file").unwrap();
+        let outfile = matches.value_of("out-file").unwrap();
+        println!("infile: {}, outfile: {}", infile, outfile);
+        let key_manager =
+            DataKeyManager::from_config(&cfg.security.encryption, &cfg.storage.data_dir)
```

Comment:
I think we need to avoid this key manager update key dictionary or file dictionary. Can you leave a `TODO` here?